test/tools/jsontests/vm.cpp

```diff
@@ -106,7 +106,7 @@ EnvInfo FakeExtVM::importEnv(mObject const& _o, LastBlockHashesFace const& _last
 	blockHeader.setDifficulty(toInt(_o.at("currentDifficulty")));
 	blockHeader.setTimestamp(toInt64(_o.at("currentTimestamp")));
 	blockHeader.setAuthor(Address(_o.at("currentCoinbase").get_str()));
-	blockHeader.setNumber(toInt(_o.at("currentNumber")));
+	blockHeader.setNumber(toInt64(_o.at("currentNumber")));
```

Comment:
Have you thought about using `toPositiveInt64` here?

Reply:
Yes, I can rework `toInt64` to `toPositiveInt64`.

Reply:
Is the "positive" good name? We want to include 0. But "nonnegative" looks weird.

Reply:
I don't have a good alternative.  We can pretend French for a moment:

> In typical French mathematical usage, zero is both positive and negative
https://math.stackexchange.com/questions/26705/is-zero-positive-or-negative